### PR TITLE
Update Copyright

### PR DESCRIPTION
**Pull request by rrsettgast**
Some cleanup for the release.
Added a proper RELEASE file.
Added RELEASE_NOTES to list modifications between releases
Updated copyright dates to 2020

@joshua-white @klevzoff Please take a look at the RELEASE_NOTES and add what you think is appropriate. After this is merged,  I'll tag it, then create a release from that tag.

NOTE. this PR is now just the copyright update. A separate PR will handle the version info into the new release branch.

### CODE REVIEW

File: RELEASE [hunk 1]
```diff
@@ -0,0 +1,11 @@
+*******************************************************************************
+
+GEOSX: ................................, version 0.2.0
```
Comment (joshua-white) [inline RELEASE:3]: why all the dots?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Version v0.2.0 -- Release date 2020-06-20
+==========================================
+* Extensive updates to Data Repository.
+* Discretizations
+  * Finite Element Interface
+  * Finite Volume Stencil
+* Physics Solvers
+  * Solid Mechanics Explcit on GPU, Implicit Assembly on GPU
+  * Single Phase Flow
+  * Composiitonal Multiphase Flow
```
Comment (joshua-white) [inline RELEASE_NOTES.md:29]: typo

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  * Surface Generation
+    * Topology change (Legacy GEOSX approach)
+    * Embedded Enhanced Assumed Strain
```
Comment (joshua-white) [inline RELEASE_NOTES.md:32]: Can we change to `Enriched Finite Element Method`

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+
+Version v0.2.0 -- Release date 2020-06-20
+==========================================
+* Extensive updates to Data Repository.
+* Discretizations
+  * Finite Element Interface
+  * Finite Volume Stencil
+* Physics Solvers
+  * Solid Mechanics Explcit on GPU, Implicit Assembly on GPU
```
Comment (joshua-white) [inline RELEASE_NOTES.md:27]: typo

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  * Single Phase Flow
+  * Composiitonal Multiphase Flow
+  * Surface Generation
+    * Topology change (Legacy GEOSX approach)
```
Comment (joshua-white) [inline RELEASE_NOTES.md:31]: Legacy GEOS approach?

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+
+Version v0.1.0 -- Release date 2018-02-15
+==========================================
+Initial Code Release containing:
+* Data Repository
+  * Group
+  * Wrapper
+  * Input processing 
+* Physics Solver hierarcy
```
Comment (joshua-white) [inline RELEASE_NOTES.md:48]: typo

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  * Surface Generation
+    * Topology change (Legacy GEOSX approach)
+    * Embedded Enhanced Assumed Strain
+  * Hydrofracture Solver
+  * Small Strain aligned Contact using Lagrange Multipliers
+* Mesh Structure
+  * Introduced concept of extrinsic mesh data
+  * Fracture Elements to represent FV cells in fracture
+* Proppant modeling 
+
```
Comment (joshua-white) [inline RELEASE_NOTES.md:39]: * VTK output
* Expanded linear algebra interfaces

File: RELEASE (hunk 1 above)
Comment (rrsettgast) [reply RELEASE:3]: what's wrong with dots? The more dots the better I always say.

**Comment by rrsettgast** [comment]
Splitting this PR up into a PR to update the copyright dates, and a new one for the release.

**Pull request merged**